tests/odh/resources/lora_train.py

```
@@ -0,0 +1,78 @@
+
+Reads the pre-downloaded model from a shared PVC, generates a small synthetic
+dataset, runs LoRA SFT training, and validates checkpoints.
+All configuration is read from environment variables.
+"""
+import json
+import os
+
+from training_hub import lora_sft
+
+MODEL_DIR = os.environ.get("MODEL_DIR", "/mnt/shared/model")
+DATA_PATH = "/tmp/train_data.jsonl"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/train_data.jsonl"

(S108)